### PR TITLE
Ignore *_example packages in coverage check

### PR DESCRIPTION
**Pull request by simahawk**
See https://github.com/OCA/website-cms/pull/9#pullrequestreview-22552007

@moylop260 alternatively we could consider looking up for custom `.coveragerc` in the repo. What do you think?

### CODE REVIEW

**Comment by lasley** [comment]
I think this change will also need a corresponding note in the contribution docs?

**Comment by simahawk** [comment]
@moylop260 @lasley there you go https://github.com/OCA/maintainer-tools/pull/263 ;)

**Comment by dreispt** [comment]
Err, sorry, I thought that the note in the docs would go in this repo.
IMO the documentation regarding MQT should go in the MQT repo

**Comment by lasley** [comment]
My thought was that _example is a convention of how to provide an example module & so we would put it in the contribution docs. Maybe both?

**Comment by simahawk** [comment]
@lasley @dreispt I added a note in MQT readme and a link to MT#tests info.

**Comment by pedrobaeza** [comment]
I'm not sure if this apply the same for Codecov. Anyway, we need to indicate the new coverage tool in the docs. @moylop260, you made the switch. Do you know if the file still applies?

**Comment by moylop260** [comment]
FYI Codecov use the same output of coverage command using the same .coveragerc file

File: README.md [hunk 1]
```diff
@@ -42,9 +42,13 @@ Coveralls configuration file
 ----------------------------
 
 Coveralls provides information on the test coverage of your modules.
-Currently the Coveralls configuration is automatic, so you don't need to include a `.coveragerc`
-to the repository. Please note that if you do it, it will be ignored.
+Currently the Coveralls configuration is automatic (check default configuration [here](cfg/.coveragerc)).
```
Comment (pedrobaeza) [inline README.md]: s/Coveralls/Codecov

Comment (simahawk) [inline README.md]: @pedrobaeza no more Coveralls? Shouldn't be "Coveralls and Codecov"?

Comment (pedrobaeza) [inline README.md]: We are extinguishing it, but we can put both in a preventive way

Comment (simahawk) [inline README.md]: updated and linked both services

**Pull request merged**